.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Release checklist
+about: Použi tento checklist na testovanie artefaktov pred public releaseom
+title: "Release "
+labels: release
+assignees: @slovensko-digital/autogram-release-team
+---
+
+## Windows
+
+- [ ] funguje inštalácia na Windows cez stiahnutý .msi
+- [ ] funguje spustenie v GUI móde
+- [ ] funguje URL handler [autogram://go](autogram://go)
+- [ ] funguje GUI otvoriť jeden súbor, ten sa zobrazí, viem ho podpísať, vytvorí sa podpísaný súbor
+
+## Linux
+- [ ] funguje inštalácia na Linux (Debian-based) cez stiahnutý .deb
+- [ ] funguje inštalácia na Linux cez stiahnutý .rpm
+- [ ] funguje spustenie v GUI móde
+- [ ] funguje URL handler [autogram://go](autogram://go)
+- [ ] funguje GUI otvoriť jeden súbor, ten sa zobrazí, viem ho podpísať, vytvorí sa podpísaný súbor
+
+## MacOS
+- [ ] funguje inštalácia na MacOS cez stiahnutý .pkg
+- [ ] funguje spustenie v GUI móde
+- [ ] funguje URL handler [autogram://go](autogram://go)
+- [ ] funguje GUI otvoriť jeden súbor, ten sa zobrazí, viem ho podpísať, vytvorí sa podpísaný súbor
+
+
+## Na aspoň jednom systéme
+
+- [ ] fungujú všetky smoke testy `./mvnw test -Psmoke`
+- [ ] funguje spustenie v GUI serverovom móde `autogram --url=autogram://listen?protocol=http&host=localhost&port=37201&origin=*&language=sk` na inom porte
```

Comment:
Ak to medzičasom nikto nedokódil, parametre host, origin a language sa nepoužívajú. Používa sa iba protocol a port